execution/executor/src/mock_vm/mod.rs

```diff
@@ -13,8 +13,8 @@ use libra_types::{
     contract_event::ContractEvent,
     event::EventKey,
     transaction::{
-        RawTransaction, Script, SignedTransaction, TransactionArgument, TransactionOutput,
-        TransactionPayload, TransactionStatus,
+        RawTransaction, Script, SignedTransaction, Transaction as ExecutorTransaction,
```

Comment:
Why this `ExecutorTransaction` alias?

Reply:
There is a `Transaction` defined at line 27

Reply:
Okay I looked at that code, maybe rename that one to `MockVMTransaction`... up to you.